sample/SkyApm.Sample.AspNet/Properties/AssemblyInfo.cs

```diff
@@ -2,35 +2,18 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
 [assembly: AssemblyTitle("SkyApm.Sample.AspNet")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("SkyApm.Sample.AspNet")]
-[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyCopyright("Copyright ©  2019")]
```

Comment:
Copyright should be 2018-2019

Reply:
this is sample proj , assembly copyright no meaning at this .

Reply:
@wu-sheng changed

Reply:
I think I am not qualified to be a reviewer :) Let @liuhaoyang @yang-xiaodong do their professional works. I was just quick searching and reviewing documents and licenses related things, to avoid potential risks for the project.